app/controllers/provinces_controller.rb

```diff
@@ -1,13 +1,17 @@
 class ProvincesController < Pumi::ProvincesController
   skip_before_action :check_guisso_cookie
+  BATTAMBANG = '02'
+  KOMPONG_CHHNANG = '04'
 
   def index
-    render(json: Pumi::ResponseSerializer.new(filter_provinces))
+    render(json: Pumi::ResponseSerializer.new(pilot_provinces))
   end
 
   private
 
-  def filter_provinces
-    Pumi::Province.where(id: "02")
+  def pilot_provinces
+    [BATTAMBANG, KOMPONG_CHHNANG].map do |id|
```

Comment:
@radinreth piloting provinces should be adding into the environment variable POLIT_PROVINCES=02,04, then create a Setting.pilot_provinces to read this from the ENV and split with comma(,) separate more than hard-coding assigned